tests/e2e/normalize.go

```diff
@@ -377,6 +379,8 @@ func normalizeHTTPResponses(t *testing.T, events test.LogEntries) {
 
 	// Compute operations
 	visitor.replacePaths[".fingerprint"] = "abcdef0123A="
+	// specific to containercluster
+	visitor.replacePaths[".startTime"] = "1970-01-01T00:00:00Z"
```

Comment:
This change is a little bit too aggressive (changed many other tests which `startTime` has already been overridden). Could you elaborate a little bit more about this change?

Reply:
The containercluster's API response has the field `startTime` that changes every time when the test tries to generate golden http log. Basically I need to normalize this field to make sure the field stay the same during each run

Reply:
Sent out https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2139 to make sure we have string replacement in script_tests

Reply:
merged https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2139 and rebased this PR.